docker-compose.override.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.2"
+services:
+  app: 
+    volumes: 
+      - /Users/jdhoffa/Documents/github/2dii/pacta-data:/pacta-data:ro
+      - /Users/jdhoffa/Desktop/pacta/input:/input:ro
+      - /Users/jdhoffa/Desktop/pacta/output:/output
```

Comment:
obviously this will need to change, @AlexAxthelm would love comments

Reply:
relative local paths?

Reply:
☝🏻 that was a suggestion

Reply:
I'd prefer to allow flexibility to the user to specify data/ input and output locations when they run `docker-compose`. Relative paths still bake in a hard dependency (that the user should have input, output and data located somewhere in the neighborhood of `PACTA_analysis`). 

Mauro had the idea to specify these in a .env file, and then run something along the lines of: 
``` bash
source .env
docker-compose up
```

but then you would need to specify, in the docker-compose file
``` yml
...
volumes: 
   - ${PACTA_DATA}:/pacta-data:ro
```

and that could also be a bit wonky.

Reply:
Allow me to elaborate why I don't want to hard-code a relative dependency. 
Let's say the user clones `PACTA_analysis`, this should be the "tool", similar to Chrome, or any other application, that allows them to "run" PACTA. They should not need to "change" anything in that tool to run it.

Imagine they also clone `pacta-data` (in this example, they have access). This should also live in some folder, and be untouched. 

Now, perhaps they might want to run PACTA on multiple portfolios, in multiple different directories. They should be able to do so, by calling `docker-compose` and specifying the path to the "input" and "output" directories of their choosing (and they should be able to run this over and over again as many times as they wish). Even the ability to specify different data paths is helpful (especially to us) if we want to point to staging data or other versions of the dataset. 

If the input/ output needs to be somewhere relative to `PACTA_analysis`, then they would need to copy their data into that folder whenever they want to run the tool, which seems like an awkward UI.

Reply:
So... I was not suggesting a hard dependency on relative paths, only that it is one possibility (so that example code in the repo doesn't need to have usernames in it, and at least hypothetically could work on a user's machine)

Reply:
Yup I understand, it was how we had initially solved this problem in `pactaCore`, but it comes with it's own host of issues (that I sorta mentioned above). 

I left my username in as an obvious flag that "this is not how this should be done and needs to be fixed". It all relates to a convo @AlexAxthelm had been having on slack on alternatives. I opened this PR exactly to demonstrate what I'm doing and try to get some feedback ;-) lol. But apologies as he has much more context that I didn't explicitly state in this PR

Reply:
I would change this file to `docker-compose.override.example.yml`, and call it good. then include instructions in the readme on how to copy it, and set your own path correctly.

If you do that, I'd also switch these volume mounts to be in long syntax, since  that will be easier for folks to change correctly (don't have to worry about stopping at `:`)